app/views/divisions/division_not_found.html.haml

```diff
@@ -0,0 +1,6 @@
+- content_for :title, 'Division not found'
+%h1= yield :title
+
+No Division found. If you entered a date, please make
```

Comment:
Huh, actually we already have a [404 page](https://github.com/openaustralia/publicwhip/blob/master/app/views/home/error_404.html.haml), [route](https://github.com/openaustralia/publicwhip/blob/master/config/routes.rb#L208), and [action](https://github.com/openaustralia/publicwhip/blob/master/app/controllers/home_controller.rb#L53). So I recon just redirect to that, and then improving that page could be a new issue/PR.